src/scrabble.js

```diff
@@ -1,6 +1,86 @@
-class Scrabble {
-  // Write your implementation here
+const letterScores = {
```

Comment:
nice use of objects to store the letter value

using toLowerCase() or toUpperCase() would have avoided having to list out the alphabet twice